gocourse05/server/analityk_sevice.go

```diff
@@ -0,0 +1,21 @@
+package server
+
+import "fmt"
+
+type AnalitykService struct {
```

Comment:
Здається, помилочка:
```suggestion
type AnaliticsService struct {
```